app/javascript/src/ChangePassword/components/ChangePassword.jsx

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import React, { useState } from 'react'
+import { createReactWrapper } from 'utilities/createReactWrapper'
+import { LoginPage, Form, ActionGroup, Button } from '@patternfly/react-core'
+import { HiddenInputs, FlashMessages } from 'LoginPage'
+import { PasswordInput } from 'ChangePassword'
+import type { FlashMessage } from 'Types'
+
+import 'LoginPage/assets/styles/loginPage.scss'
+import brandImg from 'LoginPage/assets/images/3scale_Logo_Reverse.png'
+import PF4DownstreamBG from 'LoginPage/assets/images/PF4DownstreamBG.svg'
+
+type Props = {
+  lostPasswordToken: ?string,
+  url: string,
+  errors: (?FlashMessage)[]
+}
+
+const ChangePassword = ({lostPasswordToken, url, errors}: Props) => {
+  const [password, setPassword] = useState('')
+  const [isPasswordValid, setIsPasswordValid] = useState(undefined)
+  const [passwordConfirmation, setPasswordConfirmation] = useState('')
+  const [isPasswordConfirmationValid, setIsPasswordConfirmationValid] = useState(undefined)
+  const formDisabled = !isPasswordValid || !isPasswordConfirmationValid
+
+  return (
+    <LoginPage
+      brandImgSrc={brandImg}
+      brandImgAlt='Red Hat 3scale API Management'
+      backgroundImgSrc={PF4DownstreamBG}
+      backgroundImgAlt='Red Hat 3scale API Management'
+      loginTitle='Change Password'
+      footer={null}
+    >
+      {errors && <FlashMessages flashMessages={errors}/>}
+      <Form
+        action={url}
+        noValidate
+        id='edit_user_2'
+        acceptCharset='UTF-8'
+        method='post'
+      >
+        <input type='hidden' name='_method' value='put' />
+        <HiddenInputs />
+        <PasswordInput isRequired name='password' label='Password'
+          value={password} isValid={isPasswordValid} setValue={setPassword}
+          setIsValid={setIsPasswordValid} autoFocus='autoFocus'
+        />
+        <PasswordInput isRequired name='password_confirmation' label='Password confirmation'
+          value={passwordConfirmation} isValid={isPasswordConfirmationValid}
+          setValue={setPasswordConfirmation} setIsValid={setIsPasswordConfirmationValid}
+        />
+        {lostPasswordToken &&
+          <input id='password_reset_token' type='hidden' name='password_reset_token' value={lostPasswordToken} />
+        }
+        <ActionGroup>
+          <Button className='pf-c-button pf-m-primary pf-m-block'
+            type='submit'
+            name='commit'
+            isDisabled={formDisabled}
+          >
+          Change Password
```

Comment:
This line has a wrong indentation
```suggestion
            Change Password
```